### ISSUE TITLE
make check "check-am" failing

### ISSUE THREAD

**Issue by RPGillespie6** [libuv/libuv#1341]
Getting a test failure when I run `make check` - not sure how to provide details but I am dutifully obeying the output:

```
ok 303 - fork_signal_to_child
ok 304 - fork_signal_to_child_closed
ok 305 - fork_fs_events_child
ok 306 - fork_fs_events_child_dir
ok 307 - fork_fs_events_file_parent_child
ok 308 - fork_threadpool_queue_work_simple
FAIL: test/run-tests
======================================================
1 of 1 test failed
Please report to https://github.com/libuv/libuv/issues
======================================================
Makefile:4315: recipe for target 'check-TESTS' failed
make[1]: *** [check-TESTS] Error 1
make[1]: Leaving directory '/home/user/libuv'
Makefile:4571: recipe for target 'check-am' failed
make: *** [check-am] Error 2
```

* **Version**: `1.0.0` 
* **Platform**: `Linux cobalt 4.8.0-46-generic #49-Ubuntu SMP Fri Mar 31 13:57:14 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux`

**Comment by cjihrig** [comment]
Can you try using a more recent version than `1.0.0`. Could you also specify which individual tests fail.

**Comment by RPGillespie6** [comment]
Ok, with the latest commit I get:

```bash
not ok 24 - pipe_connect_to_file
# exit code 134
# Output from process `pipe_connect_to_file`:
# Assertion failed in test/test-pipe-connect-error.c on line 53: status == UV_ENOTSOCK || status == UV_ECONNREFUSED
```

**Comment by saghul** [comment]
Please use the latest released version, 1.0.0 is waaaaay old by now. I'm closing this. Please update us if you run into the same problems with the latest release.

**Comment by RPGillespie6** [comment]
@saghul Didn't you see my second post? I said I was using the latest commit on master.
